### PR TITLE
authenticator: 4.1.6 -> 4.2.0

### LINKED ISSUE #216770: `authenticator-4.1.6`: Failed to retrieve template child
### Describe the bug
The application fails to start with the following error:

```console
[user@system:~]$ RUST_BACKTRACE=full authenticator
2023-02-17T10:02:24.141351Z  INFO authenticator::application: Authenticator (com.belmoussaoui.Authenticator)
2023-02-17T10:02:24.141373Z  INFO authenticator::application: Version: 4.1.6 ()
2023-02-17T10:02:24.141376Z  INFO authenticator::application: Datadir: /nix/store/dsb3czn6nkzqi3hs2kw38mqnnf87f3h8-authenticator-4.1.6/share/authenticator
2023-02-17T10:02:24.153104Z  INFO authenticator::models::database: Running DB Migrations...
2023-02-17T10:02:24.153269Z  INFO authenticator::models::database: Database pool initialized.
thread 'main' panicked at 'Failed to retrieve template child. Please check that it has been bound and has a #[template_child] attribute.', /build/authenticator-4.1.6-vendor.tar.gz/gtk4/src/subclass/widget.rs:1305:17
stack backtrace:
   0:     0x560e5b4ed7a9 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hc3295717f362dffd
   1:     0x560e5b52191e - core::fmt::write::h3fb106f148721ced
   2:     0x560e5b502a35 - std::io::Write::write_fmt::hcd3b3704b0c53d27
   3:     0x560e5b4ed565 - std::sys_common::backtrace::print::hbd6a7285aab48741
   4:     0x560e5b50f9bf - std::panicking::default_hook::{{closure}}::h0f72cdacd638214c
   5:     0x560e5b50f6e5 - std::panicking::default_hook::hac258dc5dca473a9
   6:     0x560e5b50ffdf - std::panicking::rust_panic_with_hook::h2e8c0695c388bdee
   7:     0x560e5b4edad2 - std::panicking::begin_panic_handler::{{closure}}::h192eca38cc2c43d2
   8:     0x560e5b4ed8fc - std::sys_common::backtrace::__rust_end_short_backtrace::ha89f9e1a3bce7ed6
   9:     0x560e5b50fb72 - rust_begin_unwind
  10:     0x560e5abe2e93 - core::panicking::panic_fmt::h35982a44ff3b9a81
  11:     0x560e5ac99d0e - <authenticator::widgets::editable_label::imp::EditableLabel as gtk4::subclass::widget::CompositeTemplate>::check_template_children::h4aa05d6091c498b8
  12:     0x560e5ae3ca82 - glib::subclass::types::instance_init::he931f6105a76f38c
  13:     0x7f6f32fb85e9 - g_type_create_instance
  14:     0x7f6f32f9bed0 - g_object_new_internal
  15:     0x7f6f32f9d5cc - g_object_new_with_properties
  16:     0x7f6f32f9e1d1 - g_object_new
  17:     0x7f6f31f0d7b9 - _gtk_builder_construct
  18:     0x7f6f31f0f335 - builder_construct
  19:     0x7f6f31f11bb2 - end_element
  20:     0x7f6f31f0f4bf - proxy_end_element
  21:     0x7f6f32117538 - _gtk_buildable_parser_replay_precompiled
  22:     0x7f6f31f1256c - _gtk_builder_parser_parse_buffer
  23:     0x7f6f31f0aee9 - gtk_builder_extend_with_template
  24:     0x7f6f320f561f - gtk_widget_init_template
  25:     0x560e5ae3c7cf - glib::subclass::types::instance_init::hbd67002040dca2ac
  26:     0x7f6f32fb85e9 - g_type_create_instance
  27:     0x7f6f32f9bed0 - g_object_new_internal
  28:     0x7f6f32f9d5cc - g_object_new_with_properties
  29:     0x7f6f32f9e1d1 - g_object_new
  30:     0x7f6f31f0d7b9 - _gtk_builder_construct
  31:     0x7f6f31f0f335 - builder_construct
  32:     0x7f6f31f11bb2 - end_element
  33:     0x7f6f31f0f4bf - proxy_end_element
  34:     0x7f6f32117538 - _gtk_buildable_parser_replay_precompiled
  35:     0x7f6f31f1256c - _gtk_builder_parser_parse_buffer
  36:     0x7f6f31f0aee9 - gtk_builder_extend_with_template
  37:     0x7f6f320f561f - gtk_widget_init_template
  38:     0x560e5ae3b84e - glib::subclass::types::instance_init::h740f1cf2717690bc
  39:     0x7f6f32fb85e9 - g_type_create_instance
  40:     0x7f6f32f9bed0 - g_object_new_internal
  41:     0x7f6f32f9d7e0 - g_object_newv
  42:     0x560e5b4cd360 - glib::object::Object::new_internal::h917aa1d7466b47fb
  43:     0x560e5b4cceba - glib::object::Object::with_type::heb34da75e7c0ccdc
  44:     0x560e5ac8841b - glib::object::Object::new::h2e55d339208ef4e8
  45:     0x560e5ad52bbf - authenticator::application::Application::create_window::hdee5d698387c3d4b
  46:     0x560e5ad18797 - <authenticator::application::imp::Application as gio::subclass::application::ApplicationImpl>::activate::h3e0717f8f0b4dacb
  47:     0x560e5ac1866d - gio::subclass::application::application_activate::h9ba6d776ffd21531
  48:     0x7f6f32f96619 - _g_closure_invoke_va
  49:     0x7f6f32fb02b7 - g_signal_emit_valist
  50:     0x7f6f32fb056f - g_signal_emit
  51:     0x7f6f31cffc18 - g_application_real_local_command_line
  52:     0x560e5ac18b2f - gio::subclass::application::application_local_command_line::h2465f4e405e16ac3
  53:     0x7f6f31cffda6 - g_application_run
  54:     0x560e5ad525c8 - authenticator::application::Application::run::h876b5a1eb152951a
  55:     0x560e5ac7cc18 - authenticator::main::h9c53bfc3a595e1a7
  56:     0x560e5adfa9e3 - std::sys_common::backtrace::__rust_begin_short_backtrace::h470d02baa4de8a33
  57:     0x560e5ac9bbb9 - std::rt::lang_start::{{closure}}::ha2d02bbbd648d76c
  58:     0x560e5b501a54 - std::rt::lang_start_internal::hb4f207ee214b8528
  59:     0x560e5ac7df05 - main
  60:     0x7f6f3142924e - __libc_start_call_main
  61:     0x7f6f31429309 - __libc_start_main_alias_1
  62:     0x560e5abe34a5 - _start
  63:                0x0 - <unknown>
```

### Steps To Reproduce
1. Try to run the application

### Expected behavior
The application should start and work properly.

### Notify maintainers

@austinbutler

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.12, NixOS, 23.05 (Stoat), 23.05.20230215.28319de`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.2`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by austinbutler**
###### Description of changes

Authenticator is currently broken, see https://github.com/NixOS/nixpkgs/issues/216770. The fix is to upgrade `gtk-rs` to >= 0.5.3.

I can't find an easy way to make Authenticator 4.1.6 work with `gtk-rs` >= 0.5.3. They've released Authenticator 4.2.0, however it relies on the development version of GTK, >= 4.9. That's why I haven't updated Authenticator in nixpkgs, I don't imagine a nixpkgs-wide bump to a development version of GTK4 would be welcomed. So instead I'm overriding GTK4 just for Authenticator.

I'm not sure what the policy is for overrides like this.

[4.1.6 -> 4.2.0 changes](https://gitlab.gnome.org/World/Authenticator/-/compare/4.1.6...4.2.0?from_project_id=4706&straight=false)

Fixes https://github.com/NixOS/nixpkgs/issues/216770.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by austinbutler** [comment]
Result of `nixpkgs-review pr 217069` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>authenticator</li>
  </ul>
</details>

**Review by SamLukeYes — APPROVED** [review]
LGTM

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -2769,7 +2769,17 @@ with pkgs;
 
   audiowaveform = callPackage ../tools/audio/audiowaveform { };
 
-  authenticator = callPackage ../applications/misc/authenticator { };
+  authenticator = callPackage ../applications/misc/authenticator {
+    # Remove when GTK is upgraded past 4.8
+    # https://github.com/NixOS/nixpkgs/issues/216770
+    gtk4 = gtk4.overrideAttrs (attrs: rec {
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:2775]:
```suggestion
    gtk4 = gtk4.overrideAttrs (_: rec {
```

**Review by SuperSandro2000 — COMMENTED** [review]
We should mark it broken for aarch64:

```
FAILED: gsk/libgsk.a.p/gl_gskglrenderjob.c.o 
gcc -Igsk/libgsk.a.p -Igsk -I../gsk -I. -I.. -Igdk/x11 -I../gdk/x11 -Igdk -I../gdk -Igtk/css -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/glib-2.0 -I/nix/store/xkkys23phqgikyvj1bas5dq6v8k7k2zg-glib-2.74.3/lib/glib-2.0/include -I/nix/store/m30r0l3k448qvpp2gxmgwl8dxhc68vvh-graphene-1.10.8-dev/include/graphene-1.0 -I/nix/store/srw9riv8d3fjjaphk5cwbpr4x7hybfzc-graphene-1.10.8/lib/graphene-1.0/include -I/nix/store/kvqzbwkylr35787v3by8fi4lfrr0yval-harfbuzz-6.0.0-dev/include/harfbuzz -I/nix/store/3j4zj8vj6cxfhgd9xf8nkbwg33kqysgz-pango-1.50.12-dev/include/pango-1.0 -I/nix/store/wls937cdx58l405vsi5njyf331smsppj-cairo-1.16.0-dev/include/cairo -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include/freetype2 -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include -I/nix/store/xsn977sb7z5b6fhbp21myy3dqldyb0r2-gdk-pixbuf-2.42.10-dev/include/gdk-pixbuf-2.0 -I/nix/store/0dmg207m9dlgdjs475mznj5x41n3rcf2-fribidi-1.0.12/include/fribidi -I/nix/store/n93wnx35cb9siqrlr7dgv0hcanc811fk-libepoxy-1.5.10-dev/include -I/nix/store/4i7hpqghp60499c6l6vxp10sak700rba-fontconfig-2.14.0-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/gio-unix-2.0 -I/nix/store/3afv6i7gp0db2s5mslc3d9c6kc18ph7g-libpng-apng-1.6.39-dev/include/libpng16 -I/nix/store/mwwvz7ia4r2nx4rc585jlmdlrcfz1rk7-libtiff-4.5.0-dev/include -I/nix/store/b5jdrbcvk2q40qvrjxqixq37vfgf8kz2-libjpeg-turbo-2.1.4-dev/include -I/nix/store/8fc3gfm0znvkfpcdfbdajnibnl0qgkyp-libxkbcommon-1.5.0-dev/include -I/nix/store/lr79z1h3ccn8fli7rmbr8i3lfy8j0lm5-wayland-1.21.0-dev/include -I/nix/store/743p97hgvrk6d3prw1q9h5h82ay8jy2q-xorgproto-2021.5/include -I/nix/store/24kqk7zpm0kpwnh17nc2qvzvzikpxmqj-libXrender-0.9.10-dev/include -I/nix/store/872j3mki34wq4701s5xmbfahsbz4j82m-libX11-1.8.3-dev/include -I/nix/store/4fbwjwn1846bbw41z24wk36dhqrprmif-libXi-1.8-dev/include -I/nix/store/arabqns9f6x8lrmbmq7ppr6mrdcxsrfv-libXext-1.3.4-dev/include -I/nix/store/65cppap8xgsvn1rawm3riz9xhi02b7yn-libXcursor-1.2.0-dev/include -I/nix/store/757xpjyd9985yaqd69clgq58ssdi4zda-libXdamage-1.1.5-dev/include -I/nix/store/4a09w9ql1hq36y6jfsvr0ly1l6lnkzid-libXfixes-6.0.0-dev/include -I/nix/store/9cac1adf9in3lh55kmm6pi7xsyw670ik-libXrandr-1.5.2-dev/include -I/nix/store/xdhjf9ns2mv54qaqmd9a6382m2dzfkzl-libXinerama-1.1.4-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -DG_LOG_USE_STRUCTURED=1 -DGLIB_DISABLE_DEPRECATION_WARNINGS '-DGTK_VERSION="4.9.4"' -D_GNU_SOURCE -fPIC -DGTK_COMPILATION '-DG_LOG_DOMAIN="Gsk"' -DG_LOG_STRUCTURED=1 -fno-strict-aliasing -Wno-missing-include-dirs -Wduplicated-branches -Wduplicated-cond -Wformat=2 -Wformat-nonliteral -Wformat-security -Wignored-qualifiers -Wimplicit-function-declaration -Wlogical-op -Wmisleading-indentation -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-prototypes -Wswitch-default -Wswitch-enum -Wundef -Wuninitialized -Wunused -Waddress -Warray-bounds -Wempty-body -Wimplicit -Wimplicit-fallthrough -Wimplicit-fallthrough=5 -Winit-self -Wint-to-pointer-cast -Wmain -Wmissing-braces -Wmissing-declarations -Wmissing-prototypes -Wnonnull -Wpointer-to-int-cast -Wredundant-decls -Wreturn-type -Wsequence-point -Wtrigraphs -Wvla -Wwrite-strings -Wcast-align -Wnull-dereference -fvisibility=hidden -MD -MQ gsk/libgsk.a.p/gl_gskglrenderjob.c.o -MF gsk/libgsk.a.p/gl_gskglrenderjob.c.o.d -o gsk/libgsk.a.p/gl_gskglrenderjob.c.o -c ../gsk/gl/gskglrenderjob.c
../gsk/gl/gskglrenderjob.c: In function 'gsk_gl_render_job_visit_texture_scale_node':
../gsk/gl/gskglrenderjob.c:3692:7: error: a label can only be part of a statement and a declaration is not a statement
 3692 |       float min_x = bounds->origin.x;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3693:7: error: expected expression before 'float'
 3693 |       float min_y = bounds->origin.y;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3695:21: error: 'min_y' undeclared (first use in this function); did you mean 'min_x'?
 3695 |       float max_y = min_y + bounds->size.height;
      |                     ^~~~~
      |                     min_x
../gsk/gl/gskglrenderjob.c:3695:21: note: each undeclared identifier is reported only once for each function it appears in
```

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (austinbutler) [reply pkgs/top-level/all-packages.nix:2775]: Changed.

**Comment by austinbutler** [comment]
Result of `nixpkgs-review pr 217069` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>authenticator</li>
  </ul>
</details>

**Comment by austinbutler** [comment]
Hmm some change in master since the last build makes it so it can't find the `gtk4` from the override.

File: pkgs/applications/misc/authenticator/default.nix [hunk 2]
```diff
@@ -80,5 +78,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Plus;
     maintainers = with lib.maintainers; [ austinbutler ];
     platforms = lib.platforms.linux;
+    broken = stdenv.isLinux && stdenv.isAarch64;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/authenticator/default.nix:81]: Please add a very short comment why it was marked broken,

Comment (austinbutler) [reply pkgs/applications/misc/authenticator/default.nix:83]: Added.

File: pkgs/applications/misc/authenticator/default.nix [hunk 3]
```diff
@@ -80,5 +78,8 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Plus;
     maintainers = with lib.maintainers; [ austinbutler ];
     platforms = lib.platforms.linux;
+    # Fails to build on aarch64 with error
+    # "a label can only be part of a statement and a declaration is not a statement"
+    broken = stdenv.isLinux && stdenv.isAarch64;
```
Comment (iclanzan) [inline pkgs/applications/misc/authenticator/default.nix:83]: I was using Authenticator on Pinephone. It’s a shame that it’s broken now.

Comment (SuperSandro2000) [reply pkgs/applications/misc/authenticator/default.nix:83]: Looks to me like an upstream bug. If you find a solution please create a PR.

```
FAILED: gsk/libgsk.a.p/gl_gskglrenderjob.c.o 
gcc -Igsk/libgsk.a.p -Igsk -I../gsk -I. -I.. -Igdk/x11 -I../gdk/x11 -Igdk -I../gdk -Igtk/css -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/glib-2.0 -I/nix/store/xkkys23phqgikyvj1bas5dq6v8k7k2zg-glib-2.74.3/lib/glib-2.0/include -I/nix/store/m30r0l3k448qvpp2gxmgwl8dxhc68vvh-graphene-1.10.8-dev/include/graphene-1.0 -I/nix/store/srw9riv8d3fjjaphk5cwbpr4x7hybfzc-graphene-1.10.8/lib/graphene-1.0/include -I/nix/store/kvqzbwkylr35787v3by8fi4lfrr0yval-harfbuzz-6.0.0-dev/include/harfbuzz -I/nix/store/3j4zj8vj6cxfhgd9xf8nkbwg33kqysgz-pango-1.50.12-dev/include/pango-1.0 -I/nix/store/wls937cdx58l405vsi5njyf331smsppj-cairo-1.16.0-dev/include/cairo -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include/freetype2 -I/nix/store/ahmvpqld8dh07wxqmv8qxl5c0wpvnbl4-freetype-2.12.1-dev/include -I/nix/store/xsn977sb7z5b6fhbp21myy3dqldyb0r2-gdk-pixbuf-2.42.10-dev/include/gdk-pixbuf-2.0 -I/nix/store/0dmg207m9dlgdjs475mznj5x41n3rcf2-fribidi-1.0.12/include/fribidi -I/nix/store/n93wnx35cb9siqrlr7dgv0hcanc811fk-libepoxy-1.5.10-dev/include -I/nix/store/4i7hpqghp60499c6l6vxp10sak700rba-fontconfig-2.14.0-dev/include -I/nix/store/hayg4ar73n9s018lqkg490imx63ffqgy-glib-2.74.3-dev/include/gio-unix-2.0 -I/nix/store/3afv6i7gp0db2s5mslc3d9c6kc18ph7g-libpng-apng-1.6.39-dev/include/libpng16 -I/nix/store/mwwvz7ia4r2nx4rc585jlmdlrcfz1rk7-libtiff-4.5.0-dev/include -I/nix/store/b5jdrbcvk2q40qvrjxqixq37vfgf8kz2-libjpeg-turbo-2.1.4-dev/include -I/nix/store/8fc3gfm0znvkfpcdfbdajnibnl0qgkyp-libxkbcommon-1.5.0-dev/include -I/nix/store/lr79z1h3ccn8fli7rmbr8i3lfy8j0lm5-wayland-1.21.0-dev/include -I/nix/store/743p97hgvrk6d3prw1q9h5h82ay8jy2q-xorgproto-2021.5/include -I/nix/store/24kqk7zpm0kpwnh17nc2qvzvzikpxmqj-libXrender-0.9.10-dev/include -I/nix/store/872j3mki34wq4701s5xmbfahsbz4j82m-libX11-1.8.3-dev/include -I/nix/store/4fbwjwn1846bbw41z24wk36dhqrprmif-libXi-1.8-dev/include -I/nix/store/arabqns9f6x8lrmbmq7ppr6mrdcxsrfv-libXext-1.3.4-dev/include -I/nix/store/65cppap8xgsvn1rawm3riz9xhi02b7yn-libXcursor-1.2.0-dev/include -I/nix/store/757xpjyd9985yaqd69clgq58ssdi4zda-libXdamage-1.1.5-dev/include -I/nix/store/4a09w9ql1hq36y6jfsvr0ly1l6lnkzid-libXfixes-6.0.0-dev/include -I/nix/store/9cac1adf9in3lh55kmm6pi7xsyw670ik-libXrandr-1.5.2-dev/include -I/nix/store/xdhjf9ns2mv54qaqmd9a6382m2dzfkzl-libXinerama-1.1.4-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -DG_LOG_USE_STRUCTURED=1 -DGLIB_DISABLE_DEPRECATION_WARNINGS '-DGTK_VERSION="4.9.4"' -D_GNU_SOURCE -fPIC -DGTK_COMPILATION '-DG_LOG_DOMAIN="Gsk"' -DG_LOG_STRUCTURED=1 -fno-strict-aliasing -Wno-missing-include-dirs -Wduplicated-branches -Wduplicated-cond -Wformat=2 -Wformat-nonliteral -Wformat-security -Wignored-qualifiers -Wimplicit-function-declaration -Wlogical-op -Wmisleading-indentation -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-prototypes -Wswitch-default -Wswitch-enum -Wundef -Wuninitialized -Wunused -Waddress -Warray-bounds -Wempty-body -Wimplicit -Wimplicit-fallthrough -Wimplicit-fallthrough=5 -Winit-self -Wint-to-pointer-cast -Wmain -Wmissing-braces -Wmissing-declarations -Wmissing-prototypes -Wnonnull -Wpointer-to-int-cast -Wredundant-decls -Wreturn-type -Wsequence-point -Wtrigraphs -Wvla -Wwrite-strings -Wcast-align -Wnull-dereference -fvisibility=hidden -MD -MQ gsk/libgsk.a.p/gl_gskglrenderjob.c.o -MF gsk/libgsk.a.p/gl_gskglrenderjob.c.o.d -o gsk/libgsk.a.p/gl_gskglrenderjob.c.o -c ../gsk/gl/gskglrenderjob.c
../gsk/gl/gskglrenderjob.c: In function 'gsk_gl_render_job_visit_texture_scale_node':
../gsk/gl/gskglrenderjob.c:3692:7: error: a label can only be part of a statement and a declaration is not a statement
 3692 |       float min_x = bounds->origin.x;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3693:7: error: expected expression before 'float'
 3693 |       float min_y = bounds->origin.y;
      |       ^~~~~
../gsk/gl/gskglrenderjob.c:3695:21: error: 'min_y' undeclared (first use in this function); did you mean 'min_x'?
 3695 |       float max_y = min_y + bounds->size.height;
      |                     ^~~~~
      |                     min_x
../gsk/gl/gskglrenderjob.c:3695:21: note: each undeclared identifier is reported only once for each function it appears in
```

Comment (iclanzan) [reply pkgs/applications/misc/authenticator/default.nix:83]: Found this merged fix in GTK:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5516

The fix is not available in gtk 4.9.4 but I tried gtk 4.10.0 and it works!

Comment (iclanzan) [reply pkgs/applications/misc/authenticator/default.nix:83]: Working on a PR at https://github.com/NixOS/nixpkgs/pull/222432.

**Pull request merged**